worlds/hades/Template/Template.yaml

```diff
@@ -0,0 +1,84 @@
+Hades:
```

Comment:
You should just remove this file altogether. The template yamls are auto-generated by AP